test/Polly.TestUtils/FakeLoggerFactory.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace Polly.TestUtils;
+
+public sealed class FakeLoggerFactory : ILoggerFactory
+{
+    public FakeLogger FakeLogger { get; } = new FakeLogger();
```

Comment:
```suggestion
    public static FakeLogger FakeLogger { get; } = new();
```

Reply:
We don't want to have static here, it should be tied to the lifetime of `FakeLoggerFactory`.

Reply:
OK, I assumed it was like `NullLogger.Instance`.